IOSum/src/Main.java

```diff
@@ -0,0 +1,39 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.stream.Collectors;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+      BufferedReader br = new BufferedReader(new FileReader("numbers1.txt"));
```

Comment:
The code is kind of correct, but the task was to implement a function, which takes a file name as a parameter and returns 0 as a sum of numbers form the file. Your code just takes a particular file and cannot be reused for another input file without changing it.